### ISSUE TITLE
Uncaught TypeError: Cannot read property 'startTrackerWithId' of undefined

### ISSUE THREAD

**Issue by teckel12** [danwilson/google-analytics-plugin#366]
Doesn't seem to be loading the Google Analytics code.  I installed using `cordova plugin add cordova-plugin-google-analytics` which ran fine.  Then, in my `init-dev.js` file I added the following:

> // Google Analytics
> window.addEventListener('deviceready', function() {
> &nbsp;&nbsp;console.log('Starting Google Analytics...');
> &nbsp;&nbsp;window.ga.startTrackerWithId('UA-XXXXXX-XX');
> &nbsp;&nbsp;window.ga.trackView('Start App');
> &nbsp;&nbsp;console.log('Google Analytics Started');
> }, true);

This doesn't seem to trigger as I never get the console log.  So, in my document.readyState I added the following:

> console.log('Starting Google Analytics...');
> window.ga.startTrackerWithId('UA-XXXXXX-XX');
> window.ga.trackView('Start App');

I then get the console error `Uncaught TypeError: Cannot read property 'startTrackerWithId' of undefined`.  It seems window.ga doesn't exist, even though it's been installed.  Am I missing a step?  It seems by the instructions I just install the plugin and add the above code and it works.  But, it doesn't seem to.

Maybe some more basic getting started instructions?  Anyway, not getting anywhere.

Thanks in advance!

**Comment by remisture** [comment]
Which platform are you on?

**Comment by OlexandrPopov** [comment]
I have the same issue. I use cordova android 6.1.1

**Comment by teckel12** [comment]
I believe I figured it out.  In your instructions you mention "In your 'deviceready' handler, set up your Analytics tracker:".  I tried setting up a deviceready event handler to start the Google tracker.  But, that wouldn't work.  Instead, I needed to add the `window.ga.startTrackerWithId` inside my dev.onDeviceReadyCordova function.  Once I did that, everything started working.

The instructions are just a bit ambiguous instead of being direct and clear.  In any case, I got screen tracking working.
